src/locales/ar.json

```diff
@@ -100,6 +100,9 @@
     }
   },
   "games": {
+    "quizGame": {
+      "title": "لعبة الاسئلة"
+    },
```

Comment:
This is not related with this pr . But only 3 lines. Probably not wort a separate PR. Maybe can be bundled with a general fixes PR.

Reply:
PTAL